### DISCUSSION TITLE
FindBySelector is always giving undefined.

### DISCUSSION THREAD

**Discussion by csruchitaoza** [clientIO/joint#2650] [Q&A]
### Introduction

I'm using jointJS in angular. I've used some custom shapes that are working fine.
But after adding a view for that shape, it gives me an error in the below line. 
    let [liquidEl] = this.**findBySelector**("liquid");

All findbySelector is giving undefined values.
trying this example - https://codepen.io/jointjs/pen/oNQpZJV/094b9fea7c7248d1e29a12ffa5b37f55.

Even though I've tried in HTML CSS js locally the same error occurs. 
Am I missing any import? 

Any help would be great.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Duplicate of https://github.com/clientIO/joint/issues/2620#issuecomment-2058433426.
